components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -247,35 +246,53 @@ public DataSection phenotypeBody(Patient patient) throws Exception
         if (sortedFeatures.size() == 0) {
             // offset is included to account for the presence of both "positive" and "negative" in "present"
             int offset = bothTypes ? 1 : 0;
-            Integer emptyX = 0;
             for (int i = 0; i < present.size() - offset; i++) {
-                DataCell cell = new DataCell("", emptyX, 0);
+                DataCell cell = new DataCell("", i, 0);
                 section.addCell(cell);
-                emptyX++;
             }
         }
         // section.finalizeToMatrix();
         return section;
     }
 
+    public void variantsSetup(Set<String> enabledFields) throws Exception
+    {
```

Comment:
Either variants should be removed from this or I don't think this PR should have been made until Variants was merged and you had rebased. A large fraction of the code people are reviewing here is irrelevant to the PR.